src/coreclr/jit/codegenarm64.cpp

```diff
@@ -10067,6 +10067,31 @@ void CodeGen::genCodeForMadd(GenTreeOp* tree)
     genProduceReg(tree);
 }
 
+//-----------------------------------------------------------------------------------
+// genCodeForMsub: Emit a msub (Multiply-Subtract) instruction
+//
+// Arguments:
+//     tree - GT_MSUB tree where op2 is GT_MUL
+//
+void CodeGen::genCodeForMsub(GenTreeOp* tree)
```

Comment:
Have you considered to just handle it as part of GT_MADD? I bet it'd be much less lines of code

Reply:
I did consider it - I guess it's a design choice whether or not to use GT_MADD or introduce GT_MSUB.

Reply:
The name GT_MADD as a lowering-only op that is specific for ARM64 which reflects the actual instruction 'madd' - at least for me, I wouldn't it to expect to emit 'msub', but I totally understand why it does though.

Reply:
> Can we consider actually just making GT_MADD only emit 'madd' 

I like this and it makes it have "parity" with `GT_ADD` and `GT_SUB`.